test/integration/http2_integration_test.cc

```diff
@@ -38,6 +38,8 @@ TEST_P(Http2IntegrationTest, MultipleContentLengths) { testMultipleContentLength
 
 TEST_P(Http2IntegrationTest, ComputedHealthCheck) { testComputedHealthCheck(); }
 
+TEST_P(Http2IntegrationTest, AddEncodedTrailers) { testAddEncodedTrailers(); }
```

Comment:
It's a weird request but mind adding this to integration_test as well?  I'd just like end to end coverage that if we add trailers over-enthusiastically on the upstream or downstream side that we still get a complete response.

Reply:
not weird at all! i was going to but it slipped my mind

Reply:
this actually caught a bug, so it was a great idea